### PR TITLE
Feature/css about

### PR DESCRIPTION
**Pull request by AIbrahimQA**

### CODE REVIEW

File: app/views/about.scala.html [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+        <center>
+            <h2 class=" qa_header"> About Us   </h2>
+            <div class="border"></div>
+        </center>
+        <center>
+            <br>
 
-        <h3> Team 3  </h3>
+            <p class="about_header"> <b>Why Choose Us?</b> </p>
+            <p class="about_para" We provide a variety of up-to-date films with different classifications to choose from which are shown on state of the art screens.<br>
```
Comment (DannyPennington) [inline app/views/about.scala.html]:
```suggestion
            <p class="about_para"> We provide a variety of up-to-date films with different classifications to choose from which are shown on state of the art screens.<br>
```

Comment (DannyPennington) [reply app/views/about.scala.html]: Some of the text doesn't show on the page, I think this might be why

File: app/views/about.scala.html [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
+            <H4 class="about_header"> Sam White  </H4>
+            <a href="https://github.com/SamWhite745" class="card-link" target="_blank">Link to Github</a>
+            <br>
+            <H4 class="about_header"> Taiwo Oluwatolafun Aina-Badejo   </H4>
+            <a href="https://github.com/Tola12343" class="card-link" target="_blank">Link to Github</a>
+            <br>
+            <br>
+            <p class="about_para"> <b> For more information a link to the contact page can be found below:</b></p>
+            <br>
+            <a class = "btn-lg btn-light" href="http://localhost:9000/contactUs" style="padding: 20px;">Contact Page</a>
```
Comment (DannyPennington) [inline app/views/about.scala.html]:
```suggestion
            <a class = "btn-lg btn-light" href="@routes.ContactUsController.contact" style="padding: 20px;">Contact Page</a>
```

Comment (DannyPennington) [reply app/views/about.scala.html]: Changed to reverse route over hard-coded link

**Review by DannyPennington — CHANGES_REQUESTED** [review]
Nice work, just some little edits

**Review by DannyPennington — APPROVED** [review]

**Review by SamWhite745 — APPROVED** [review]

**Pull request merged**